app/controllers/teams_controller.rb

```diff
@@ -70,7 +69,8 @@ def update_params_with_current_user
     if current_user.admin?
       team_params
     else
-      team_params.merge('user_ids': (team_params[:user_ids] << current_user.id.to_s))
+      user_ids = team_params[:user_ids] || []
+      team_params.merge('user_ids': (user_ids << current_user.id.to_s))
```

Comment:
nit: seems weird to me to quote the key (I'd expect `user_ids:` instead of `'user_ids':`

Reply:
Agree, willfix